<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>biblioteq_greyliteratureinfo</class>
 <widget class="QMainWindow" name="biblioteq_greyliteratureinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BiblioteQ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/icons.qrc">
    <normaloff>:/book.png</normaloff>:/book.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <layout class="QHBoxLayout">
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/16x16/ok.png</normaloff>:/16x16/ok.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="resetButton">
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Reset</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="printButton">
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Print...</string>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/16x16/fileprint.png</normaloff>:/16x16/fileprint.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Close</string>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/16x16/cancel.png</normaloff>:/16x16/cancel.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>printButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
